### PR TITLE
[RM Ch07] Section 7.7 SBOM

### LINKED ISSUE #2542: [RM Ch07] Elaborate on SBOM
Provide guidance on standards for communicating SBOM information

### PR DESCRIPTION
**Pull request by karinesevilla**
Fixes #2542 
Content on SPDX and SWID tags

### CODE REVIEW

**Comment by jbhartley** [comment]
@walterkozlowski @karinesevilla Sorry Walter & Karine, though I have reviewed and would approve if able, I do not have approval access.

**Comment by walterkozlowski** [comment]
@pgoyal01, @ulikleber : could you please review and approve if you can?  Thank you.

File: doc/ref_model/chapters/chapter07.md
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter07.md:420]: Typo: duplicate "format"
```suggestion
A transparent software supply chain offers benefits for vulnerabilities remediation, but also for licensing management and it provides assurance of the source and integrity of components. In order to use the SBOM efficiently and spread its adoption, information must be generated and shared in a standard format. This format must be machine-readable to allow automation. The NTIA white paper [“Survey of existing SBOM formats and standards”](https://www.ntia.doc.gov/files/ntia/publications/ntia_sbom_formats_and_standards_whitepaper_-_version_20191025.pdf) identifies two key formats covering baseline SBOM information: Software Package Data eXchange (SPX) and Software Identification Tags (SWID tags).
```

File: doc/ref_model/chapters/chapter07.md
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter07.md:424]: Typo
```suggestion
[SWID Tags](https://nvd.nist.gov/products/swid) is an international XML-based standard used by commercial software publishers and has been published as the standard ISO/IEC 19770-2. The specification defines four types of SWID tags: primary, patch, corpus, and supplemental to describe a software component.
```

**Review by pgoyal01 — APPROVED** [review]
subject to minor typo corrections

**Comment by pgoyal01** [comment]
@jbhartley You should be able to review now.

**Comment by pgoyal01** [comment]
@wmk-admin You need to add @jbhartley as a member and maybe remove others including from owners.

**Comment by walterkozlowski** [comment]
@jbhartley, I can see that Pankaj approved this PR 2 hours ago.

**Comment by walterkozlowski** [comment]
@karinesevilla, I think you did not reposed to suggestions by @pgoyal01 (some small editorial improvements).  Once you do this, @jbhartley is happy to approve as well, and I will be able to merge.  Thanks!

**Pull request merged**
